contracts/libraries/VaultLifecycleSTETH.sol

```diff
@@ -146,28 +135,29 @@ library VaultLifecycleSTETH {
         )
     {
         uint256 pendingAmount = uint256(vaultState.totalPending);
-        uint256 roundStartBalance = currentBalance.sub(pendingAmount);
-
-        uint256 singleShare = 10**uint256(vaultParams.decimals);
+        uint256 decimals = vaultParams.decimals;
```

Comment:
We need to rename this variable to `_decimals` instead because `decimals` shadows the `decimals()` function